lib/paypalhttp/http_client.js

```diff
@@ -53,7 +53,11 @@ class HttpClient {
 
     Object.keys(headers).forEach(function (key) {
       if (key != null) {
-        formattedHeader[key.toLowerCase()] = headers[key];
+        if (key.toLowerCase() === 'content-type')	{
+          formattedHeader[key.toLowerCase()] = headers[key].toLowerCase();
+        } else {
+          formattedHeader[key.toLowerCase()] = headers[key];
+        }
```

Comment:
Can you please explain the reason behind segregating the statement into a if else block?